cmake/FindFastjet.cmake

```diff
@@ -0,0 +1,134 @@
+#----------------------------------------------------------------------
```

Comment:
Is this cmake file used? 
We probably want our dependency to fast jet optional.